.travis.yml

```diff
@@ -47,8 +47,7 @@ before_install:
 install: true
 
 script:
-  - mvn clean test verify -B
-  #- ./core/files/travis-build.sh
+  - mvn -DfailIfNoTests=false clean test verify -B
```

Comment:
The argument `-DfailIfNoTests=false` is necessary because the lgpl tools module has no tests